.github/workflows/ci.yml

```diff
@@ -30,6 +30,11 @@ jobs:
           ansible-ansible-language-server-*.tgz
 
   test:
+    name: >-
+      NodeJS ${{ matrix.node-version }} / ${{ matrix.os }}${{
+        fromJSON(matrix.experimental) && ' [XFAIL]' || ''
+      }}
```

Comment:
This is not what I would call short, simple. Please use something like:
```
node16-macos
node16-windows-xfail
```

As I told you yesterday, if it would up to me I would even remove node matrix testing as we really need only the minimal version (12). If we **really** want we could add one job that tests latest node on a single platform.

My opinion is that CI jobs should only be added when there is a proof (bug) that current ones failed to catch. Reference to other projects is not relevant for us.

Reply:
I still want to keep the job names human-readable.